### PR TITLE
[MSE] Add flow combiner to handle non-linked branches

### PR DESCRIPTION
**Pull request by calvaris**
inputselector can end up having non-active that end up being
non-linked branches and we need the flow combiner to deal with those.

### CODE REVIEW

File: Source/WebCore/platform/graphics/gstreamer/mse/WebKitMediaSourceGStreamerPrivate.h [hunk 1]
```diff
@@ -122,6 +122,8 @@ struct _WebKitMediaSrcPrivate {
 
     GRefPtr<GstBus> bus;
     WebCore::MediaPlayerPrivateGStreamerMSE* mediaPlayerPrivate;
+
+    GUniquePtr<GstFlowCombiner> flowCombiner;
```
Comment (eocanha) [inline Source/WebCore/platform/graphics/gstreamer/mse/WebKitMediaSourceGStreamerPrivate.h]: I would #include <wtf/glib/GUniquePtr.h> in this file. It's true that there's another pre-existing GUniquePtr above, but it's currently working by chance and the proper thing to do is to include the header.

**Comment by thiblahute** [comment]
lgtm, FlowCombiner should handle seeks and flush properly as it is intensively used in the demuxers but yeah, I guess we will see.

Should be proposed upstream?

File: Source/WebCore/platform/graphics/gstreamer/mse/WebKitMediaSourceGStreamerPrivate.h (hunk 1 above)
Comment (calvaris) [reply Source/WebCore/platform/graphics/gstreamer/mse/WebKitMediaSourceGStreamerPrivate.h]: No, it is included in GStreamerCommon.h. It's a change I did some time ago.

Comment (calvaris) [reply Source/WebCore/platform/graphics/gstreamer/mse/WebKitMediaSourceGStreamerPrivate.h]: Actually I added both GUniquePtrGStreamer and GRefPtrGStreamer to that file and ensured it was declared in all GStreamer files.

**Pull request merged**